src/brs/props/CaselessProperties.java

```diff
@@ -0,0 +1,34 @@
+package brs.props;
+
+import java.util.Properties;
+
+/**
+ * Overrides {@link java.util.Properties} to make all the keys lowercase,
+ * thus making the config file keys case-insensitve.
+ * Courtesy Jim Yingst
+ * https://coderanch.com/t/277128/java/Properties-ignoring-case
+ */
+public class CaselessProperties extends Properties {
```

Comment:
I'd have called it `CaseInsentiveProperties` - but you know that naming is hard...

Reply:
Haha I can rename it in a future pr. I just used what the source called it.